app/views/finance/ecf/duplicates/compare/_induction_record_summary_list.html.erb

```diff
@@ -0,0 +1,38 @@
+<%= govuk_summary_list do |summary_list|
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "ID" }
+    row.value  { induction_record.id }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Lead provider" }
+    row.value  { induction_record.lead_provider.name }
+  end
+  summary_list.row(classes: "govuk-body-s") do |row|
+    row.key { "Cohort" }
+    row.value { induction_record.cohort.start_year.to_s }
```

Comment:
```suggestion
    row.value { induction_record.cohort.display_name }
```

Reply:
@cwrw I'd be reluctant to use this method as it's coupling the view layer with the database layer. I think we should ✂️ this method